src/components/UniversalNavbarExpanded/DropdownNav/DropdownCta.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// Reused assets from UniversalNavbar
+import { DropdownLinkIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { Spacer, Body, COLORS, TitleSmall } from '../../index'
+
+// Helpers
+import { removeLastWord, getLastWord } from '../../../helpers/splitLastWord'
+
+// Styles
+import styles from './DropdownCta.module.scss'
+
+/**
+ * CTA copy for use in DropdownNav
+ *
+ *
+ * @param {string} title - A short title
```

Comment:
I've found that styleguidist actually documents these better if we slightly depart from jsdoc convention (like we have here), and then move down to the PropTypes defs (I'll leave another comment down there)